assets/css/main.css

```diff
@@ -0,0 +1,579 @@
+@import "fontello.css";
+
+:root {
+  --measure: 80ch;
+
+  /* space 1 */
+  --s1: 2rem;
+
+  /* space 2 */
+  --s2: 10px;
+
+  --background: #f6f7f8;
+  --darker-background: #dadee2;
+  --darkest-background: #8694a1;
+
+  --text-color: #495561;
+
+  --primary: #27a8e0;
```

Comment:
@fabientownsend recommends to decouple the names of the CSS vars from the actual colors names, but more for their intent, as described here: https://material.io/design/color/the-color-system.html#color-theme-creation